app/components/Layout/Header.tsx

```diff
@@ -80,19 +92,64 @@ const HeaderLayout = ({
             {isLoggedIn ? (
               <>
                 {isRegistered && (
-                  <li>
-                    <Link href="/user/profile">
-                      <a className="nav-button">Profile</a>
-                    </Link>
-                  </li>
+                  <>
+                    <li className="d-none d-lg-block">
+                      <Dropdown alignRight>
+                        <Dropdown.Toggle variant="primary">
+                          <FontAwesomeIcon
+                            color="white"
+                            icon={faUser}
+                            size="1x"
+                          />
```

Comment:
Could we find a way to have the user icon button appear the same (circular background, inverse colors - dropdown caret would be the only difference) as on narrower viewports?